### PR TITLE
Add library QC types

### PR DESCRIPTION
**Pull request by apmasell**
Add the ability to do certain QC procedures in bulk via the bulk library edit page. This is a temporary feature since it doesn't allow editing existing QCs in bulk nor setting dates or other particulars on QC. This is a stop gap measure to allow adding bulk QCs for OICR rollout until we can design something more comprehensive.

### CODE REVIEW

**Comment by frankie13** [comment]
Description here would be so useful for me!  Or maybe a link back to the ticket whence this feature came?

**Comment by apmasell** [comment]
Updated.

File: miso-web/src/main/webapp/scripts/library_hot.js [hunk 1]
```diff
@@ -644,6 +668,9 @@ Library.hot = {
 
       lib.dnaSize = obj.dnaSize;
       lib.volume = obj.volume;
+      lib.qcQubit = obj.qcQubit;
+      lib.qcTapeStation = obj.qcTapeStation;
+      lib.qcQPcr = obj.qcQPcr;
```
Comment (callunity) [inline miso-web/src/main/webapp/scripts/library_hot.js]: Please wrap these in stupid checks.

Comment (apmasell) [inline miso-web/src/main/webapp/scripts/library_hot.js]: Fixed.

File: miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryRestController.java [hunk 2]
```diff
@@ -134,6 +140,24 @@ public void setLibraryService(LibraryService libraryService) {
       throw new RestException("No such library.", Status.NOT_FOUND);
     }
     library = Dtos.to(libraryDto);
+    if (libraryDto.getQcQubit() != null) {
+      LibraryQC qc = new LibraryQCImpl();
+      qc.setQcType(requestManager.getLibraryQcTypeByName("Qubit"));
```
Comment (callunity) [inline miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryRestController.java]: Hmm good (sad) point.

**Pull request merged**